PlaceRouteHierFlow/run_small_in_docker.sh

```diff
@@ -15,7 +15,7 @@ lcov -z ; \
 (cd cap_placer && ./unit_tests --gtest_output=xml:junit.xml) && \
 (cd placer && ./unit_tests --gtest_output=xml:junit.xml) && \
 (cd router && ./unit_tests --gtest_output=xml:junit.xml) && \
-./pnr_compiler ./testcase_example switched_capacitor_filter.lef switched_capacitor_filter.v switched_capacitor_filter.map layers.json switched_capacitor_filter 2 0 > PnR.log ; \
+./pnr_compiler ./testcase_example switched_capacitor_filter.lef switched_capacitor_filter.v switched_capacitor_filter.map FinFET_Mock_PDK_Abstraction.json switched_capacitor_filter 2 0 > PnR.log && \
```

Comment:
Not sure why the pull request does not show a merge conflict. Feels like the filename should be layers.json